plugins/Quotes/class.quotes.plugin.php

```diff
@@ -544,6 +541,20 @@ protected function formatQuote($type, $id, &$quoteData, $format = false) {
         Emoji::instance()->enabled = $emojiEnabled;
     }
 
+    /**
+     * Checks if the user can view discussion.
+     *
+     * @param object $discussion
+     * @throws Exception If the user cannot view the discussion.
+     */
+    private function canViewDiscussion(object $discussion): void {
+        $userID = Gdn::session()->UserID;
+        $canView = $this->discussionModel->canView($discussion, $userID);
+        $isAdmin = Gdn::session()->checkRankedPermission('Garden.Moderation.Manage');
```

Comment:
`$isAdmin` for moderator permissions?

Reply:
it's a ranked permission. The user could be an admin.

Reply:
They could be. But that's not what we're testing for, specifically. This code reads like "Are they an admin? Check to see if they have moderator permissions."